framework/src/mesh/MooseMesh.C

```diff
@@ -486,19 +486,21 @@ MooseMesh::prepare(const MeshBase * const mesh_to_clone)
   // This is done *after* the side boundaries (e.g. "right", ...) have been generated.
   if (isParamValid("add_sideset_ids") && !isParamValid("add_sideset_names"))
   {
-    const auto & add_sideset_ids = getParam<std::vector<BoundaryID>>("add_sideset_ids");
-    _mesh_boundary_ids.insert(add_sideset_ids.begin(), add_sideset_ids.end());
-    _mesh_sideset_ids.insert(add_sideset_ids.begin(), add_sideset_ids.end());
+    const auto & add_boundary_ids = getParam<std::vector<BoundaryID>>("add_sideset_ids");
```

Comment:
the parameter is still called sideset so there is still confusion between sidesets and ndoesets here

Reply:
let's simply add the "add_nodeset_ids/names" capability

Reply:
I think that's the right way of doing it. Side and node sets are fundamentally different, and we should offer separate parameters for them.

Reply:
I've added a lambda that handles the sidesets/nodesets.